api/Parser.cpp

```diff
@@ -517,6 +495,32 @@ ConferenceEndpointExtendedInfo parseEndpointExtendedInfo(const nlohmann::json& d
     return endpoint;
 }
 
+EndpointDescription::Ice parseIce(const nlohmann::json& iceJson)
+{
+    api::EndpointDescription::Ice ice;
+    ice._ufrag = iceJson["ufrag"].get<std::string>();
+    ice._pwd = iceJson["pwd"].get<std::string>();
+
+    for (const auto& candidateJson : iceJson["candidates"])
```

Comment:
Please make it optional like it was before. Candidates are not really required

```suggestion
    for (const auto& candidateJson : optionalJsonArray(iceJson, "candidates"))
```